src/pages/dev/amplifier/chain-integration/governance-proposals.mdx

```diff
@@ -0,0 +1,341 @@
+    --gas auto --gas-adjustment 1.5 --gas-prices 0.007uamplifier \
+    --chain-id=devnet-amplifier \
+    --no-admin \
+    --node $RPC
+```
+</tab-item>
+
+<tab-item title="Non-EVM integrators">
+Once the feedback period has passed, submit the proposal with the [proper instantiation parameters](https://github.com/axelarnetwork/axelar-amplifier/blob/deab56a75209a158c082cbf9b77caf500eb4ec7c/contracts/gateway/src/msg.rs#L4):
+
+```bash
+export NETWORK_GOV_ADDRESS="axelar10d07y265gmmuvt4z0w9aw880jnsr700j7v9daj"
+export RPC="http://devnet-amplifier.axelar.dev:26657"
+```
+
+```bash
+axelard tx gov submit-proposal store-instantiate [path to contract as wasm binary or gzip] \
+    '{
+        "verifier_address": "axelar1e6jnuljng6aljk0tjct6f0hl9tye6l0n9p067pwx2374h82dmr0s9qcqy9",
+        "router_address": "axelar14jjdxqhuxk803e9pq64w4fgf385y86xxhkpzswe9crmu6vxycezst0zq8y"
+    }' \
+    --title="Store and instantiate gateway contract for [chain name]" \
+    --description="Store and instantiate gateway contract for [chain name]" \
+    --run-as $NETWORK_GOV_ADDRESS \
+    --label "[chain name]" \
+    --deposit=2000000000uamplifier \
+    --keyring-backend test \
+    --from wallet \
+    --gas auto --gas-adjustment 1.5 --gas-prices 0.007uamplifier \
+    --chain-id=devnet-amplifier \
+    --no-admin \
+    --node $RPC
+```
+</tab-item>
+
+</tabs>
+
+## Proposal to instantiate the prover contract
+
+<tabs>
+
+<tab-item title="EVM integrators">
+If you are integrating an EVM chain, you can submit a proposal to instantiate the pre-deployed prover contract with the [proper instantiation parameters](https://github.com/axelarnetwork/axelar-amplifier/blob/deab56a75209a158c082cbf9b77caf500eb4ec7c/contracts/multisig-prover/src/msg.rs#L12) using its existing `code_id` (`495`):
+
+```bash
+export CODE_ID=495
+export CHAIN_NAME="[chain name]"
+export ADMIN_ADDRESS="axelar1zlr7e5qf3sz7yf890rkh9tcnu87234k6k7ytd9"
+export NETWORK_GOV_ADDRESS="axelar10d07y265gmmuvt4z0w9aw880jnsr700j7v9daj"
+export RPC="http://devnet-amplifier.axelar.dev:26657"
+```
+
+```bash
+axelard tx gov submit-proposal instantiate-contract $CODE_ID \
+    '{
+"admin_address": $ADMIN_ADDRESS,
+"governance_address": "axelar1zlr7e5qf3sz7yf890rkh9tcnu87234k6k7ytd9",
+"gateway_address": "0xFe34259F1EEc1Ea702A175bC5C2a4a29df6cfb43", # Fantom source gateway address
+"multisig_address": "axelar19jxy26z0qnnspa45y5nru0l5rmy9d637z5km2ndjxthfxf5qaswst9290r",
+"coordinator_address":"axelar1m2498n4h2tskcsmssjnzswl5e6eflmqnh487ds47yxyu6y5h4zuqr9zk4g",
+"service_registry_address":"axelar1c9fkszt5lq34vvvlat3fxj6yv7ejtqapz04e97vtc9m5z9cwnamq8zjlhz",
+"voting_verifier_address": "0xFe34259F1EEc1Ea702A175bC5C2a4a29df6cfb43", # Fantom voting verifier source gateway address
+"signing_threshold": ["1","1"],
+"service_name": "validators",
+"chain_name": "[chain name]",
+"verifier_set_diff_threshold": 1,
+"encoder": "abi",
+"key_type": "ecdsa",
+"domain_separator": [hash($CHAIN_NAME, $ADMIN_ADDRESS, 505)]
```

Comment:
is hash a real function?

Reply:
It's submitted as a string in a hex format (I tested with `""domain_separator": "6973c72935604464b28827141b0a463af8e3487616de69c5aa0c785392c9fb9f"`). Clarified in the code sample.